frontend/src/views/CollectionView.vue

```diff
@@ -0,0 +1,40 @@
+<template>
+  <div class="container-fluid vh-100">
+    <div class="row">
+      <div class="col col-3 sticky-top sidebar">
+        <SideBarComponent />
+      </div>
+      <div class="col content d-flex flex-column justify-content-between">
+        <div>
+          <ProjectView v-if="hasActivatedProject" />
+        </div>
+        <FooterComponent />
+      </div>
+    </div>
+  </div>
+</template>
+
+<script setup lang="ts">
+import SideBarComponent from '@/components/SideBarComponent.vue'
+import FooterComponent from '@/components/FooterComponent.vue'
+import ProjectView from './ProjectView.vue'
+import { useProjectIdStore } from '@/stores/stores'
+import { computed } from 'vue'
+const storeProjectId = useProjectIdStore()
+const hasActivatedProject = computed(() => storeProjectId.projectId != null)
```

Comment:
_ActivatedProject_, what's this? Aren't we just expecting a valid project ID? If so `hasValidProjectId` seems more appropriate